packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -28,7 +28,7 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
     where: {
       slug: slug,
       // If this is under an org, check that the team doesn't already exist
-      ...(isOrgChildTeam && { parentId: user.organizationId }),
+      parentId: isOrgChildTeam ? user.organizationId : null,
```

Comment:
Also needed to check parent null explicitly when on org is in context, otherwise it could select wrongly